templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +28,12 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">UPDATE</a>
+              </td>
+              <td>
+                <a href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id  %}">DELETE</a>
```

Comment:
Ensure there are no extra spaces within the URL tag `{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}`. Remove any unnecessary spaces to avoid potential issues.